### ISSUE TITLE
change import path github.com/influxdb/influxdb to github.com/influxdata/influxdb

### ISSUE THREAD

**Issue by oiooj** [influxdata/influxdb#5388]
``` golang

import (
	"time"

	"github.com/influxdb/influxdb/toml"
)

```
should change all `github.com/influxdb/influxdb` to `github.com/influxdata/influxdb` in import path.

or go build will use old modules

**Comment by hc000** [comment]
I noticed alot of files still reference github.com/influxdb/influxdb instead of github.com/influxdata/influxdb

**Comment by zcalusic** [comment]
Yeah, same problem here, won't compile due to bad imports.

Here's a simple fix that works well for me:
```perl
perl -pi -e 's%github.com/influxdb/%github.com/influxdata/%' `find . -name \*.go`
```

That command should be run in $GOPATH/src/github.com/influxdata/influxdb, be careful!

**Comment by slimsag** [comment]
Is there any opposition towards fixing this? Is it just as simple as changing the import paths and sending a PR?

**Comment by jwilder** [comment]
@slimsag We're holding off on fixing this until after `0.10` release.

**Comment by toddboom** [comment]
@slimsag if you change your local repo back to `influxdb/influxdb`, this should work without issue in the meantime.

**Comment by slimsag** [comment]
@jwilder Got it, thanks! Looks like `0.10` is has just three issues open, so it shouldn't be too long a wait I hope.

@toddboom Are you saying to change my import path back to `github.com/influxdb/influxdb`? I assume that works because of GitHub redirect?

My motivation for wanting this fixed sometime soon, is that we'd like to incorporate InfluxDB as the storage backend of choice for Appdash (see https://github.com/sourcegraph/appdash/pull/99) and as such won't be able to do so unless influxdb is go-gettable (we will be embedding it).

**Comment by toddboom** [comment]
@slimsag If your on-disk path is `$GOPATH/src/github.com/influxdb/influxdb` it should work with the import paths unchanged. We should have `v0.10.0` released by the end of the week, and then we'll move towards making the import path switch internally after that. The folder rename should work fine until then - let me know if it doesn't!

**Comment by zcalusic** [comment]
No, it's not working due to internal packages.
```
package github.com/influxdb/influxdb/cmd/influx_tsm/tsdb
	imports github.com/influxdata/influxdb/cmd/influx_tsm/tsdb/internal: use of internal package not allowed
```

I wonder why the tree is left in inconsistent state for such a long time, when a fix is so simple? Whoever renamed repository should've just renamed all the imports at the same time. It's really not a rocket science.

**Comment by corylanou** [comment]
This has been prioritized and will be fixed soon. We have a large branch that needs to be merged and then this fix is next.  To avoid a big merge conflict we held off on making the change.  I agree the change itself is simple, but the rebase/merge could potentially be messy.  I know this has caused some pain for the community.

**Comment by zcalusic** [comment]
OK, makes sense, good look with the big merge!

Here's another tip then, to survive until then. Instead of only git pull, you can do:
```
git checkout -f
git pull
perl -pi -e 's%github.com/influxdb/%github.com/influxdata/%' `find $GOPATH/src/github.com/influxdata/influxdb -name *.go`
```

That way you can update & have the tree in compilable state.

**Comment by chris-ramon** [comment]
just heads-up you might want to close this issue, fixes on #5617 - btw thanks a lot for fixing this! :star2:

**Comment by mark-rushakoff** [comment]
Closed by #5617.
